packages/babel-plugin-transform-modules-commonjs/test/fixtures/misc/import-const-throw/output.js

```diff
@@ -18,24 +18,32 @@ _baz.Baz = (44, function () {
   throw new Error('"' + "Baz" + '" is read-only.');
 }());
 ({
-  Foo: _foo.default
-} = {});
+  Foo
+} = ({}, function () {
+  throw new Error('"' + "Foo" + '" is read-only.');
+}()));
```

Comment:
This change is correct. You can see the input file at https://github.com/babel/babel/blob/93ec6ee62a10ea022c902962dc989f1045fb1637/packages/babel-plugin-transform-modules-commonjs/test/fixtures/misc/import-const-throw/input.mjs

```js
import Foo from "foo";

// ...

({Foo} = {});
```